src/sinks/loki.rs

```diff
@@ -88,7 +104,12 @@ impl SinkConfig for LokiConfig {
             return Err("`labels` must include at least one label.".into());
         }
 
-        let request_settings = self.request.unwrap_with(&TowerRequestConfig::default());
+        if !self.request.concurrency.is_none() {
```

Comment:
Nit: I might do `if self.request.concurrency.is_some()` to avoid the "double negative".